src/code42cli/cmds/securitydata.py

```diff
@@ -160,6 +156,26 @@ def _get_saved_search_query(ctx, param, arg):
     cls=incompatible_with("advanced_query"),
 )
 
+begin_option = opt.begin_option(
+    SECURITY_DATA_KEYWORD,
+    callback=lambda ctx, param, arg: convert_datetime_to_timestamp(
+        limit_date_range(arg, max_days_back=90)
```

Comment:
`90` should probably be a constant defined somewhere so it's easy to change

Reply:
I don't know if adding another variable makes it any easier to change, the first place I'd go to look for the config for this option would be at the option definition, making it a constant somewhere else just means I have to make one more hop to see what it is/change it.